src/scripts/main.js

```diff
@@ -354,7 +354,23 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const calculateAge = (born, died) => died - born;
+const calculateCentury = (year) => Math.ceil(year / 100);
```

Comment:
The `calculateCentury` function correctly calculates the century by dividing the year by 100 and using `Math.ceil` to round up. This is a standard approach for determining centuries.